DESCRIPTION

```diff
@@ -2,7 +2,7 @@ Type: Package
 Package: EpiNow2
 Title: Estimate Real-Time Case Counts and Time-Varying
     Epidemiological Parameters
-Version: 1.3.6.2000
+Version: 1.3.6.2001
```

Comment:
Dev version needs to go up in increments of a 1000.

Reply:
This has been fixed in b853c987.